packages/cashier-v2/src/lib/Transfer/utils/utils.tsx

```diff
@@ -0,0 +1,17 @@
+import { getMT5AccountDetails } from '../../../helpers';
+import { TTransferableAccounts } from '../types';
+
+export const getTransferAccountName = (account: TTransferableAccounts[number]) => {
```

Comment:
Same here, the display names should be normalized throughout the application and it should be coming from BE.

Reply:
This is removed.